hazelcast/src/main/java/com/hazelcast/internal/diagnostics/HealthMonitor.java

```diff
@@ -274,8 +274,8 @@ private void logDiagnosticsHint() {
         private double memoryUsedOfMaxPercentage;
 
         public void update() {
-            memoryUsedOfTotalPercentage = PERCENTAGE_MULTIPLIER * runtimeUsedMemory.read() / runtimeTotalMemory.read();
-            memoryUsedOfMaxPercentage = PERCENTAGE_MULTIPLIER * runtimeUsedMemory.read() / runtimeMaxMemory.read();
+            memoryUsedOfTotalPercentage = (PERCENTAGE_MULTIPLIER * runtimeUsedMemory.read()) / runtimeTotalMemory.read();
```

Comment:
it's not not related to this very change, but can't `runtimeTotalMemory` ever return 0?